test/internal/helpers/ingress.go

```diff
@@ -0,0 +1,27 @@
+package helpers
+
+import (
+	"fmt"
+
+	"github.com/kong/kubernetes-testing-framework/pkg/clusters"
+	netv1 "k8s.io/api/networking/v1"
+	netv1beta1 "k8s.io/api/networking/v1beta1"
+	"k8s.io/apimachinery/pkg/runtime"
+)
+
+// -----------------------------------------------------------------------------
+// Ingress Helpers
+// -----------------------------------------------------------------------------
```

Comment:
I've been wondering how these "section" comments are useful. They typically are used when the content below them should be extracted as a separate file IMHO. In this case it's already a separate file and the comment duplicates the package/file pattern. Do you mind removing it?

Reply:
👍